BookingSystem/BookingDatabase/Script.PostDeployment.sql

```diff
@@ -0,0 +1,163 @@
+﻿/*
+Post-Deployment Script Template							
+--------------------------------------------------------------------------------------
+ This file contains SQL statements that will be appended to the build script.		
+ Use SQLCMD syntax to include a file in the post-deployment script.			
+ Example:      :r .\myfile.sql								
+ Use SQLCMD syntax to reference a variable in the post-deployment script.		
+ Example:      :setvar TableName MyTable							
+               SELECT * FROM [$(TableName)]					
+--------------------------------------------------------------------------------------
+*/
+
+
+IF (EXISTS(SELECT * FROM [dbo].[Users]))
+BEGIN
+    DELETE FROM [dbo].[Users]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[Roles]))
+BEGIN
+    DELETE FROM [dbo].[Roles]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[Bookings]))
+BEGIN
+    DELETE FROM [dbo].[Bookings]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[BookingExtraServices]))
+BEGIN
+    DELETE FROM [dbo].[BookingExtraServices]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[ExtraServices]))
+BEGIN
+    DELETE FROM [dbo].[ExtraServices]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[RoomNumbers]))
+BEGIN
+    DELETE FROM [dbo].[RoomNumbers]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[Rooms]))
+BEGIN
+    DELETE FROM [dbo].[Rooms]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[Hotels]))
+BEGIN
+    DELETE FROM [dbo].[Hotels]
+END
+
+
+IF (EXISTS(SELECT * FROM [dbo].[Locations]))
+BEGIN
+    DELETE FROM [dbo].[Locations]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[Cities]))
+BEGIN
+    DELETE FROM [dbo].[Cities]
+END
+
+IF (EXISTS(SELECT * FROM [dbo].[Countries]))
+BEGIN
+    DELETE FROM [dbo].[Countries]
+END
+
+
+INSERT INTO [Users]
+			([FirstName], [SecondName], [Email], [PasswordHash], [PasswordSalt])
+VALUES
+	('admin', 'admin', 'admin@admin.com', '1', '1'),
+	('user', 'user', 'user@user.com', '2', '1');
+
+
+INSERT INTO Roles ([Name])
+VALUES
+	('Admin'),
+	('User');
+
+
+
+
+INSERT INTO [dbo].[Countries]([Name]) 
+VALUES ('United States'), ('United Kingdom'), ('Spain')
+
+INSERT INTO [dbo].[Cities]([CountryId], [Name])
+SELECT [Countries].[CountryId], [Cities].[Name]
+FROM (SELECT [CountryId], ROW_NUMBER() OVER (ORDER BY (SELECT 0)) AS row_num 
+	  FROM [dbo].[Countries]) AS [Countries]
+JOIN 
+	(SELECT [Name], ROW_NUMBER() OVER (ORDER BY (SELECT 0)) AS row_num
+	 FROM (VALUES ('New York'),('London'),('Madrid')) AS [Cities]([Name])) AS [Cities]
+ON [Countries].row_num = [Cities].row_num
+
+
+INSERT INTO [dbo].[Hotels]([Name],[Address],[CountryId], [CityId])
```

Comment:
This is a wrong place for test data. You should insert here some predefined data like Counties, Cities, Roles. Please move everything you will use for test purpose somewhere else, for example, to Manual Scripts